test/test_honeywellbackend.py

```diff
@@ -41,7 +42,6 @@
 from .decorators import online_test
 
 
-@online_test
```

Comment:
I don't really understand why the skip decorator is moved from the class, nothing in the class is testable unless there are credentials available. But it doesn't really change things I guess. My bigger concern here is that almost all the testing in here requires access to the HQS. We really should setup CI with a test account so we can run these tests.

Reply:
When it was decorating the class it just didn't work (it would always fail)...  It is a function decorator, so it can really only wrap a function.  I can create a class decorator to handle wrapping the class, but it seemed more straight-forward to just wrap the two functions instead.

Reply:
If we can provide credentials, but would prefer to avoid committing them to the repository itself, is there a good way to provide them to the build environment? (e.g. as environment variables)

Reply:
Yeah, both azure pipelines and travis support encrypted secrets stores (as does github actions which is on my todo to migrate this repo to). They don't work for jobs from PR but do for PRs from branches on this repo and for merges. That's what I would use for these testing jobs I definitely wouldn't want to commit credentials to the repo (or if we did we'd have to encrypt them with ssl).

Reply:
If you can provide an e-mail address we can go through the account setup for that account to get you access to an API validation endpoint.  It allows for submitting arbitrary QASM programs and returns '0' results in the correct format.